playbooks/main.yml

```diff
@@ -61,3 +61,6 @@
 
 - import_playbook: ocp-iscsi.yml
   when: ocp_iSCSI_enabled
+
+- import_playbook: request-header-identity-provider.yml
+  when: request_header_provider
```

Comment:
If the condition is `when: request_header_provider is defined and request_header_provider` it will no longer be a required param. This also helps with backward compatibility.

I have seen it in the other PRs and would like to see it changed.

Reply:
Changed the condition to `when: request_header_provider is defined and request_header_provider `